edwardpizarro-loops-fatima.js

```diff
@@ -0,0 +1,144 @@
+//Question 1
+//Write a for loop that will log only the even numbers in 0 through 200.
+// Hint: Think about the increment expression.
+
+
+for(var i = 0; i <= 200; i+=2) { // will add plus 2
+    console.log(i)
+}
+
+
+//Question 2
+//Write code that logs "Love me, pet me! HSSSSSS!" 10 times.
+
+// For every even number in your loop, log "...human...why you taking pictures of me?...", "...the catnip made me do it...", "meow?", or "...why does the red dot always get away..." at random.
+
+// Hint: You will need to use Math.random()
+
+   kittens = ['human why you take pictures','the catnip made me do it', 'meow?', 'why does the red dot always get away']
+   kittens[0];
+   kittens[1];
+   kittens[2];
+  for (let i = 0; i < 10; i++) {
+    if (i % 2 === 0) {
+    let randomChoice = Math.floor(Math.random() * 4) 
+    console.log(kittens[randomChoice])
+    }
+  }
+//I wasnt sure on how to get that code to print 10 times. I also had a classmate help me on this is so shout out to thst person. But for the mos part I do get whats going on.
```

Comment:
it isn't printing 10 times because your condition tells it to print ONLY on the even-numbered loops. try adding other conditions that print things when the loop is odd!